app/src/main/java/com/bpn/trainme/MyApplication.kt

```diff
@@ -2,6 +2,15 @@ package com.bpn.trainme
 
 import android.app.Application
 import dagger.hilt.android.HiltAndroidApp
+import timber.log.Timber
 
 @HiltAndroidApp
-class MyApplication : Application()
\ No newline at end of file
+class MyApplication : Application(){
+
+    override fun onCreate() {
+        super.onCreate()
+        if(BuildConfig.DEBUG){
+            Timber.plant(Timber.DebugTree())
+        }
```

Comment:
Looks like the code isn't formatted. I'd apply the standard formatting of the code. Maybe helpful to learn the code format keyboard shortcut (CMD/CTRL + ALT + L)